Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,76 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-558-ga70c9ba-src.tar.gz"
+  sha256 "a07cb6cc92bb7f05cc35f738bbde627b655df62ac7dbcd272716581c7b6bf5b1"
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "9999"
+  end
+
+  head do
+    url "https://github.com/tarantool/tarantool.git", :branch => "master", :shallow => false
+    version "1.6.*"
+  end
+
+  option "with-debug", "Build Debug version"
+  option "with-test", "Run Tests after building"
```

Comment:
If these tests don’t take too long to run we can enable it everytime and remove the option.